app/scripts/controllers/network/fake-block-tracker.ts

```diff
@@ -0,0 +1,11 @@
+import { PollingBlockTracker } from 'eth-block-tracker';
+
+/**
+ * Implements just enough of the block tracker interface to pass the tests but
```

Comment:
Nit: This suggests that this mock implements a subset of the block tracker API. But instead it appears that this block tracker _is a real, complete block tracker_ with a single method overridden.

Reply:
You're right, this description is inaccurate. It's also not super clear how overriding `eventNames` prevents the block tracker from running (assuming it does). I've fixed both of these issues: [6a1e7c3](https://github.com/MetaMask/metamask-extension/pull/18628/commits/6a1e7c3f985e38f237621fe6c024df6aa15b8607)